src/Client.php

```diff
@@ -513,6 +513,63 @@ public function upsertUser(array $user): CallResult
         );
     }
 
+    public function deleteUser(array $user): CallResult
+    {
+        $payload = [
+            "identification" => $this->userIdentifiersToArray(
+                new UserIdentified(
+                    isset($user["userId"]) ? (string) $user["userId"] : null,
+                    isset($user["email"]) ? (string) $user["email"] : null
+                )
+            ),
+        ];
+
+        $body = $this->streamFactory->createStream(json_encode($payload));
+
+        $response = $this->http->sendRequest(
+            $this->withAuthentication(
+                $this->withUserAgent(
+                    $this->requestFactory
+                        ->createRequest(
+                            "DELETE",
+                            new Uri("{$this->rootUrl}/users")
+                        )
+                        ->withHeader("content-type", "application/json")
+                        ->withBody($body)
+                )
+            )
+        );
+
+        $result = $this->check($response);
+
+        if ($result) {
+            return $result;
+        }
+
+        $response->getBody()->rewind();
+        $json = json_decode($response->getBody()->getContents(), true);
+
+        if ($response->getStatusCode() === 201) {
```

Comment:
```suggestion
        if ($response->getStatusCode() === 202) {
```

We'll need to change this ^^